sdk/formrecognizer/Azure.AI.FormRecognizer/src/RecognizedForm.cs

```diff
@@ -104,6 +104,12 @@ internal RecognizedForm(string formType, FormPageRange pageRange, IReadOnlyDicti
         /// </summary>
         public IReadOnlyList<FormPage> Pages { get; }
 
+        /// <inheritdoc/>
+        public override string ToString()
+        {
+            return $"FormType: {FormType}, # Fields: {Fields.Count}, # Pages: {Pages.Count}";
```

Comment:
I intentionally didn't include a guideline about adding properties that have `Id` on the name or equivalent like we discussed internally as it might not apply to all types. Instead focus on differentiate instances
In this case, `ModelId` can be null so not included in the ToString